modules/express/test/unit/routeHandlers.ts

```diff
@@ -0,0 +1,82 @@
+// eslint-disable-next-line
+/// <reference types="mocha" />
+import * as should from 'should';
+
+import 'should-http';
+import 'should-sinon';
+import '../lib/asserts';
+
+import * as nock from 'nock';
+import { Environments } from 'bitgo';
+import { handleV2ConsolidateAccount } from '../../src/clientRoutes';
+
+const expressApp = require('../../src/expressApp').app;
+const request = require('supertest-as-promised');
+
+const { BitGo } = require('bitgo');
+
+const oldFetchConstants = BitGo.prototype.fetchConstants;
```

Comment:
Could you leave a comment here explaining why we need to monkey patch the bitgo prototype?